cucumber/src/main/java/foundation/stack/datamill/cucumber/CommandLineSteps.java

```diff
@@ -60,7 +60,7 @@ public void executeCommandExpectingFailure(String command) {
         executeCommandExpectingFailureFromRelativeLocation(command, null);
     }
 
-    @When("^" + Phrases.SUBJECT + "( \"(.+)\")* executes \"([^\"]+)\" from \"(.+)\", it should fail$")
+    @When("^" + Phrases.SUBJECT + " executes \"([^\"]+)\" from \"(.+)\", it should fail$")
```

Comment:
I added it because I thought it could be useful to denote which user was actually performing the step, but realised for these cases it wasn't needed..